setup.py

```diff
@@ -32,7 +32,6 @@
         "Programming Language :: Python :: Implementation :: PyPy",
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3 :: Only",
-        "Programming Language :: Python :: 3.6",
```

Comment:
Can you please add a `python_requires` too? It's sad we missed that part as Python 2 users will get non-working installs by default.

Reply:
Good point. I've added `python_requires` and changed `pypy3` to `pypy-3.7` in ci.

Reply:
Actually we haven't released the Python 2 removal change yet, so Python 2.7 and Python 3.6 will get the previous version when we released this one, possibly as 1.0.

Can you update trustme/newsfragments/346.removal.rst? Sorry that I missed that

Reply:
No worries!

@pquentin What about this:
`Remove support for Python < 3.7. trustme now requires Python>=3.7 (CPython or PyPy).`

As the #346 was for dropping Python 2 support and we are going to drop Python 3.6 in this PR, should we change the file name to `416.removal.rst`?

Reply:
Ok, I've updated it!